src/radio/index.js

```diff
@@ -148,6 +151,16 @@ class Radio extends EventEmitter {
     this.emit('skips', this.skips, needed);
   }
 
+  async addSongWrapper(link: string, user: Discord.User) {
+    const passedUrl = new URL(link);
+    if (passedUrl.pathname === '/playlist') {
+      return playlistParser(String(passedUrl.searchParams.get('list'))).then(items =>
```

Comment:
<sub>honestly this `String()` is really gross but i know it's flowppression</sub>